src/platform/datapath_raw_xdp_win.c

```diff
@@ -165,32 +165,37 @@ CxPlatGetRssQueueProcessors(
     _Out_writes_to_(*Count, *Count) uint32_t* Queues
     )
 {
+    UNREFERENCED_PARAMETER(Xdp);
     uint32_t TxRingSize = 1;
     XDP_TX_PACKET TxPacket = { 0 };
     CreateNoOpEthernetPacket(&TxPacket);
 
     for (uint16_t i = 0; i < *Count; ++i) {
         HANDLE TxXsk = NULL;
-        QUIC_STATUS Status = Xdp->XdpApi->XskCreate(&TxXsk);
+        QUIC_STATUS Status = XskCreate(&TxXsk);
         if (QUIC_FAILED(Status)) { return Status; }
 
         XSK_UMEM_REG TxUmem = {0};
+        UINT32 EnableAffinity = 1;
         TxUmem.Address = &TxPacket;
         TxUmem.ChunkSize = sizeof(XDP_TX_PACKET);
         TxUmem.Headroom = FIELD_OFFSET(XDP_TX_PACKET, FrameBuffer);
         TxUmem.TotalSize = sizeof(XDP_TX_PACKET);
 
-        Status = Xdp->XdpApi->XskSetSockopt(TxXsk, XSK_SOCKOPT_UMEM_REG, &TxUmem, sizeof(TxUmem));
+        Status = XskSetSockopt(TxXsk, XSK_SOCKOPT_UMEM_REG, &TxUmem, sizeof(TxUmem));
         if (QUIC_FAILED(Status)) { CloseHandle(TxXsk); return Status; }
 
-        Status = Xdp->XdpApi->XskSetSockopt(TxXsk, XSK_SOCKOPT_TX_RING_SIZE, &TxRingSize, sizeof(TxRingSize));
+        Status = XskSetSockopt(TxXsk, XSK_SOCKOPT_TX_RING_SIZE, &TxRingSize, sizeof(TxRingSize));
         if (QUIC_FAILED(Status)) { CloseHandle(TxXsk); return Status; }
 
-        Status = Xdp->XdpApi->XskSetSockopt(TxXsk, XSK_SOCKOPT_TX_COMPLETION_RING_SIZE, &TxRingSize, sizeof(TxRingSize));
+        Status = XskSetSockopt(TxXsk, XSK_SOCKOPT_TX_COMPLETION_RING_SIZE, &TxRingSize, sizeof(TxRingSize));
+        if (QUIC_FAILED(Status)) { CloseHandle(TxXsk); return Status; }
+
+        Status = XskSetSockopt(TxXsk, XSK_SOCKOPT_TX_PROCESSOR_AFFINITY, &EnableAffinity, sizeof(EnableAffinity));
```

Comment:
What's the impact of calling this on the current official release of XDP? I believe this is new, and therefore will fail. I'm wondering if we can ignore 'NOT SUPPORTED' error codes to make this handle down-level (at least for now) better.

Reply:
Yes, please do not require this be supported.